types/expo/tsconfig.json

```diff
@@ -15,8 +15,8 @@
         "noEmit": true,
         "forceConsistentCasingInFileNames": true
     },
-    "files": [
+    "include": [
```

Comment:
It seems that using `include` instead of "files" is forbidden by the "DefinitelyTyped" rules.
The ci is failing because of https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/458172437#L460, which come from the package [types-publisher](https://github.com/Microsoft/types-publisher/blob/master/src/lib/definition-parser.ts#L216) which is
> the source code for the types-publisher service, which publishes the contents of DefinitelyTyped to npm.

Reply:
Dang. Thanks a lot for looking into this. I’ll update the PR later today.